src/Governance.ts

```diff
@@ -46,4 +49,35 @@ export class Governance {
   public async createProposal(args: CreateProposalParams): Promise<TransactionQueue<Proposal>> {
     return createProposal.prepare(args, this.context);
   }
+
+  /**
+   * Get the minimum amount of POLYX to be used as a deposit for create a public referendum proposal
```

Comment:
Get the minimum amount of POLYX that has to be deposited when creating a proposal